web-registry/src/pages/Dashboard/MyCreditClasses/MyCreditClasses.tsx

```diff
@@ -17,11 +26,14 @@ export const MyCreditClasses = (): JSX.Element => {
     <>
       <Grid container spacing={8}>
         <Grid item xs={12} md={6} lg={4}>
-          <CreateCreditClassCard
-            isFirstCreditClass={isFirstCreditClass}
-            onClick={handleCreate}
-          />
+          {isCreditClassCreator && (
+            <CreateCreditClassCard
+              isFirstCreditClass={isFirstCreditClass}
+              onClick={handleCreate}
+            />
+          )}
           {/* TODO: display user credits */}
+          {isCreditClassAdmin && <p>credit class admin credits todo...</p>}
```

Comment:
if the person is a credit class admin, we will show a list of credits that the person is an admin for. i believe that we are going to open a separate issue to handle this todo. i will update here when that's decided